store/kv/fdb.go

```diff
@@ -229,6 +229,24 @@ func (d *fdbkv) DropTable(ctx context.Context, name []byte) error {
 	return nil
 }
 
+// TableSize calculates approximate table size in bytes
+// It also works with the prefix of the table name,
+// allowing to calculate sizes of multiple table with the same prefix
+func (d *fdbkv) TableSize(ctx context.Context, name []byte) (int64, error) {
+	s := subspace.FromBytes(name)
+
+	var sz int64
+	_, err := d.txWithRetry(ctx, func(tr fdb.Transaction) (interface{}, error) {
+		var err error
+		sz, err = tr.GetEstimatedRangeSizeBytes(s).Get()
+		return nil, err
+	})
+
+	log.Err(err).Str("name", string(name)).Int64("size", sz).Msg("table size")
```

Comment:
if err is nil then we don't want to log it? Or move it to debug log if err is nil.

Reply:
ok. fixing.
there is no functionality in zerolog to only log on error.